cluster/gce/gci/configure-helper.sh

```diff
@@ -1806,6 +1806,7 @@ function start-kube-controller-manager {
   params+=" --kubeconfig=/etc/srv/kubernetes/kube-controller-manager/kubeconfig"
   params+=" --root-ca-file=${CA_CERT_BUNDLE_PATH}"
   params+=" --service-account-private-key-file=${SERVICEACCOUNT_KEY_PATH}"
+  params+="--volume-host-allow-local-loopback=false"
```

Comment:
missing a space.

@cheftako which pr do you want to merge?

Reply:
Done thanks.

Note that the k/k/cluster change seems to be necessary for the current GKE setup, at least according to my testing.

Reply:
Happy to merge either. Will go ahead and approve this one.
/approve

Reply:
let's merge both :)

Reply:
I was referring to https://github.com/kubernetes/cloud-provider-gcp/pull/183